.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,3 +1,19 @@
+## In plain terms
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)